src/logger.js

```diff
@@ -44,7 +44,14 @@ export type LoggerType = {|
 |};
 
 function httpTransport({ url, method, headers, json } : {| url : string, method : string, headers : { [string] : string }, json : Object |}) : ZalgoPromise<void> {
-    return request({ url, method, headers, json }).then(noop);
+    const hasHeaders = headers && headers.length;
```

Comment:
Headers is an object right? So probably need `headers && Object.keys(headers).length`